test/content/support.js

```diff
@@ -202,39 +202,21 @@ function resetDB() {
  * Equivalent to JSON.stringify, except that object properties are stringified
  * in a sorted order.
  */
-function stableStringify(obj, level, label) {
-	if (!level) level = 0;
-	let indent = '\t'.repeat(level);
-	
-	if (label) label = JSON.stringify('' + label) + ': ';
-	else label = '';
-	
-	if (typeof obj == 'function' || obj === undefined) return null;
-	
-	if (typeof obj != 'object' || obj === null) return indent + label + JSON.stringify(obj);
-	
-	if (Array.isArray(obj)) {
-		let str = indent + label + '[';
-		for (let i=0; i<obj.length; i++) {
-			let json = stableStringify(obj[i], level + 1);
-			if (json === null) json = indent + '\tnull'; // function
-			str += '\n' + json + (i < obj.length-1 ? ',' : '');
+function stableStringify(obj) {
+	function recompose (obj) {
+		var o = {},
+			keys = Object.keys(obj).sort();
+		for (let i=0,ilen=keys.length;i<ilen;i++) {
+			o[keys[i]] = obj[keys[i]];
 		}
-		return str + (obj.length ? '\n' + indent : '') + ']';
+		return o;
 	}
-	
-	let keys = Object.keys(obj).sort(),
-		empty = true,
-		str = indent + label + '{';
-	for (let i=0; i<keys.length; i++) {
-		let json = stableStringify(obj[keys[i]], level + 1, keys[i]);
-		if (json === null) continue; // function
-		
-		empty = false;
-		str += '\n' + json + (i < keys.length-1 ? ',' : '');
-	}
-	
-	return str + (!empty ? '\n' + indent : '') + '}';
+	return JSON.stringify(obj, function(k, v){
+		if ("object" === typeof v && !Array.isArray(v)) {
+			v = recompose(v);
```

Comment:
Since you're only calling it once, I'd get rid of the `recompose` function and move the contents here. Cleaner and more efficient.

Other than that, looks good to me.

Can you also regenerate tests (`-g` flag) and see if anything changes?